src/main/java/net/dries007/tfc/CommonEventHandler.java

```diff
@@ -807,6 +810,30 @@ public static void pickupQuiverItems(EntityItemPickupEvent event) //only pickups
         }
     }
 
+    @SubscribeEvent
+    public static void onEntityInteract(PlayerInteractEvent.EntityInteract event)
+    {
+        ResourceLocation entityType = EntityList.getKey(event.getTarget());
+        Entity target = event.getTarget();
+        EntityPlayer player = event.getEntityPlayer();
+
+        for (String entityName : ConfigTFC.GENERAL.pluckableEntities)
+        {
+            if (!event.getTarget().getEntityWorld().isRemote && player.getHeldItemMainhand().isEmpty() && player.isSneaking())
+            {
+                if (target.hurtResistantTime == 0)
+                {
+                    target.dropItem(Items.FEATHER, 1);
+                    target.attackEntityFrom(PLUCKING, (float) ConfigTFC.GENERAL.damagePerFeather);
+                }
+                if (target instanceof EntityAnimalTFC)
+                {
+                    ((EntityAnimalTFC) target).setFamiliarity(((EntityAnimalTFC) target).getFamiliarity() - 0.4f);
```

Comment:
If this is the intention you can mark this as resolved, but 0.4f = 40% of the total familiarity. Shouldn't it be 4% instead?

Reply:
Correct. Bad conversion from classic math to %. Should be .04!